### PR TITLE
fix bug 1201958 - Revert commit eef61169effdf51ec799e116735d3058110d104c

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by jezdez** [comment]
There is no need to extract the summary on every request, please move this into a new denormalized field as we have on the `Document` model. That probably means adjusting the naming of the fields involved since the other "summary" is really something about SEO and this one about "text" summary (?). Or maybe call the other one "SEO text" and this "summary"?

**Comment by groovecoder** [comment]
I can work on that and update the PR. I suggest we don't block on it though, since this is a regression for minimal performance gain.

**Comment by jezdez** [comment]
@groovecoder Sure, if you can make the blank exception more specific I'd be okay letting this be optimized later.

**Comment by stephaniehobson** [comment]
Summary and zones are behaving as expected again!! 

R+ from me. Assigning to @jezdez to check that the blank exception is being handled properly.

File: kuma/wiki/helpers.py [hunk 1]
```diff
@@ -139,6 +141,25 @@ def wiki_bleach(val):
...
+@register.filter
+def selector_content_find(document, selector):
+    """
+    Provided a selector, returns the relevant content from the document
+    """
+    content = ''
+    try:
+        page = pq(document.rendered_html)
+    except ValueError:
+        # pass errors during constuction
```
Comment (jezdez) [inline kuma/wiki/helpers.py]: minor typo

**Comment by jezdez** [comment]
rwc+

**Pull request merged**